src/main/java/mezz/jei/transfer/BasicRecipeTransferHandlerServer.java

```diff
@@ -53,10 +50,19 @@ public static void setItems(EntityPlayer player, Map<Integer, Integer> slotIdMap
 			return;
 		}
 
```

Comment:
When removing items, either as sets or as individual items, the results are now tracked in a map. The values are copies of the item stacks with their counts each set to how many were successfully removed from the container.

Reply:
I tried to leave as much of the original logic in place as possible.